Makefile

```diff
@@ -7,6 +7,7 @@ AUTODELETE_DATASET_BUCKET_FILE=True
 LOG_LEVEL=INFO
 PROJECT_ID = $(shell gcloud config get project)
 API_URL:=http://localhost:3000
+DATASET_TOPIC_ID=ons-sds-dataset-events
```

Comment:
This should be more generic to the event please such as "ons-sds-publish-dataset-event"

Reply:
Will update

Reply:
@tidmanmike Updated this